### PR TITLE
Add Song to Playlist Test

### PR DESCRIPTION
**Pull request by bryansills**
blah

### CODE REVIEW

**Comment by ejohnson44** [comment]
Looks good!

File: addsongtoplaylisttest.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env monkeyrunner
+'''
+Copyright (C) 2012  Diego Torres Milano
```
Comment (reidbaker) [inline addsongtoplaylisttest.py:3]: We cannot pull this code in unless we get the authors permission or find his licence. This is a blocking issue.

File: addsongtoplaylisttest.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env monkeyrunner
```
Comment (reidbaker) [inline addsongtoplaylisttest.py:1]: This looks like a lot of code we dont need. After we figure out the copyright issue if we can use it lets pair it down to just what we are using.

**Comment by dgreenhalgh** [comment]
Yeah, Sills, this is all around not okay.

I told Chip that we had these tests in place based on trust that these would be done. Please let me know when you can have this done, which needs to be ASAP because we're already past that deadline.

**Pull request closed without merging**